tendrl/node_agent/node_sync/services_and_index_sync.py

```diff
@@ -15,6 +15,7 @@
     "etcd",
     "tendrl-api",
     "tendrl-gluster-integration",
+    "tendrl-monitoring-integration",
```

Comment:
flag is added in common definition file to assign flag but without checking service it won't add